src/Symfony/Component/Security/Core/Resources/translations/security.tl.xlf

```diff
@@ -52,19 +52,27 @@
             </trans-unit>
             <trans-unit id="13">
                 <source>Account has expired.</source>
-                <target>Ang akawnt ay nag-expire na.</target>
+                <target>Ang account ay nag-expire na.</target>
```

Comment:
Oh, is this the same as in English?

Reply:
Yeah, for historical reasons Tagalog often doesn't have equivalents for a lot of the words used today. I've seen people "tagalogize" some words, like otel (hotel) or nars (nurse), but given the country's close ties to America, people just use English words in lieu of Tagalog ones and they're universally understood.

Actually, people often prefer to use English words even if a Tagalog equivalent exists. I would probably get strange looks in Manila for asking where the "aklatan" is instead of the "library". :D